src/modules/ekf2/EKF/output_predictor.h

```diff
@@ -150,17 +151,31 @@ class OutputPredictor
 		matrix::Quatf    quat_nominal{1.f, 0.f, 0.f, 0.f}; ///< nominal quaternion describing vehicle attitude
 		matrix::Vector3f vel{0.f, 0.f, 0.f};               ///< NED velocity estimate in earth frame (m/sec)
 		matrix::Vector3f pos{0.f, 0.f, 0.f};               ///< NED position estimate in earth frame (m/sec)
-	};
 
-	struct outputVert {
-		uint64_t time_us{0};          ///< timestamp of the measurement (uSec)
-		float    vert_vel{0.f};       ///< Vertical velocity calculated using alternative algorithm (m/sec)
-		float    vert_vel_integ{0.f}; ///< Integral of vertical velocity (m)
-		float    dt{0.f};             ///< delta time (sec)
+		float vert_vel_alt{0.f};                           ///< Vertical velocity calculated using alternative algorithm (m/sec)
+		float vert_vel_integ{0.f};                         ///< Integral of vertical velocity (m)
+		float dt{0.f};                                     ///< delta time (sec)
```

Comment:
do we use that anywhere?

Reply:
Yes, it's primarily used for the alternate velocity (z_deriv), but it's also slipped in a few places for checks (accumulated dt in new output before pushed into output buffer).

https://github.com/PX4/PX4-Autopilot/blob/e38718fde9ee2032b81d2b9e09751e348d30ece4/src/modules/ekf2/EKF/output_predictor.cpp#L381-L383